playbooks/roles/jenkins/tasks/jenkins.yml

```diff
@@ -31,7 +31,7 @@
             backup=no
 
 - name: Ensure Jenkins is installed.
-  package: name=jenkins state=present
+  package: name=jenkins state=latest
```

Comment:
Can we stipulate a major version that `latest` can't go above? The Ansible docs aren't particularly helpful on this...

Reply:
interesting....i'll have a look

Reply:
It can be done!

Reply:
Excellent 👍